libs/shared/data-context/src/lib/shared-data-context.tsx

```diff
@@ -1,18 +1,18 @@
-import { createContext, useState, useCallback } from 'react';
+import { createContext, ReactNode, useState } from 'react';
 
-type CounterContextType = {
+export type CounterContextType = {
   counter: number;
   increment: () => void;
   decrement: () => void;
 };
 
 export const DataContext = createContext<CounterContextType>({
   counter: 1,
-  increment: () => {},
-  decrement: () => {},
+  increment: () => null,
```

Comment:
`() => {}` is a function with empty body. It's better for it to return something (for example `null` )